Hydra/Serializer/CollectionNormalizer.php

```diff
@@ -92,7 +100,7 @@ public function normalize($object, $format = null, array $context = array())
                 '@context' => $this->contextBuilder->getResourceContext($resource, $context),
                 '@id' => $context['request_uri'],
             ];
-            list($parts, $parameters) = $this->parseRequestUri($context['request_uri']);
+            $uri = $this->parseRequestUri($context['request_uri']);
```

Comment:
Of course I can always revert the changes here and just do this at [L176](https://github.com/dunglas/DunglasApiBundle/blob/2636fe6830890b330fa2d80e403e4bbf4230240f/Hydra/Serializer/CollectionNormalizer.php#L176):

```php
$query = http_build_query($parameters, '', '&', PHP_QUERY_RFC3986);
$parts['query'] = preg_replace('/%5B[0-9]+%5D/', '%5B%5D', $query);
```